pkg/querier/queryrange/codec.go

```diff
@@ -1594,7 +1608,7 @@ func NewEmptyResponse(r queryrangebase.Request) (queryrangebase.Response, error)
 			},
 		}, nil
 	case *logproto.IndexStatsRequest:
-		return &logproto.IndexStatsResponse{}, nil
+		return &IndexStatsResponse{}, nil
```

Comment:
this change feels unrelated?

Reply:
poked around with this a bit, I see this change was needed to make the return type compatible with the new interface methods. 

I suspect this change is actually fixing a small mistake/oversight as well.